### ISSUE TITLE
Running cross-validation manually (for nnUNet v1)

### ISSUE THREAD

**Issue by marinbenc** [MIC-DKFZ/nnUNet#1399]
Hello,

First of all I would like to thank you for all of your hard work. nnUNet is a huge contribution to the research community!

I'm working on a project evaluating demographic bias mitigation in medical image segmentation, and I'd like to reproduce nnUNet's results in a couple of challenges. Namely, I'm trying to run the pretrained KiTS2021 model using nnUNet v1.

I have downloaded the pretrained model and am able to run `nnUNet_predict` successfully, however I'm wondering if there is a way to run the predictions for each fold, using only the validation set in each fold? I.e. the way it is run at the end of training.

I tried to look over the codebase but I didn't find a way to run the cross validation, except for writing a script manually. Can you either point me in the right direction, or if I need to write the script myself maybe give me a couple of pointers so I don't mess things up and end up validating on the train set.

I appreciate your time!

**Comment by FabianIsensee** [comment]
Hey, thanks for your warm words :-) it is great to hear the work is appreciated!
If you wanted to rerun the validation then you need to 
- `nnUNet_plan_and_preprocess` the dataset
- download and extract the pretrained weights
- `nnUNet_train 3d_fullres nnUNetTrainerV2 TASK_ID 0 --val`

This is because nnU-Net does the validation using the already preprocessed data.
Alternatively you can use the split to copy the image files to separate folders, one for each fold and then run `nnUNet_predict [...] -f X` where X is the fold with which you want to do the inference. 

Best,
Fabian

**Comment by marinbenc** [comment]
I have completely missed the `val` option 🤦. Thank you for pointing me in the right direction. :)
